createdisk.sh

```diff
@@ -47,6 +47,10 @@ ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- sudo podman pull quay.io/crcont
 # Stop the kubelet service so it will not reprovision the pods
 ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- sudo systemctl stop kubelet
 
+# Check if system reserved block exist in kubelet config
```

Comment:
It seems this is used to detect some buggy situation when the kubelet config file was empty? Would be useful to mention this in the comment rather than repeating what the line below already says :)